tests/handlers/filesystem/test_squashfs.py

```diff
@@ -0,0 +1,184 @@
+00000060  00 00 00 00 00 00 00 16  01 00 00 00 00 00 00 34  |...............4|
+00000070  01 00 00 00 00 00 00 78  da 4b 2c 28 c8 49 35 e4  |.......x.K,(.I5.|
+00000080  4a 04 51 46 10 ca 18 42  99 70 01 00 8b ee 09 3b  |J.QF...B.p.....;|
+00000090  3b 00 78 da 73 92 62 f8  7f 71 33 73 12 33 03 26  |;.x.s.b..q3s.3.&|
+000000a0  60 07 62 27 a8 3c 0b 16  39 64 79 56 34 79 3e 34  |`.b'.<..9dyV4y>4|
+000000b0  79 26 34 79 51 a8 fc 45  79 86 ff d7 81 f2 8c 40  |y&4yQ..Ey......@|
+000000c0  36 48 8d 07 54 9e 0d 88  01 cc 3c 11 94 37 00 78  |6H..T.....<..7.x|
+000000d0  da 63 66 00 02 66 10 e1  c0 c9 c0 90 58 50 90 93  |.cf..f......XP..|
+000000e0  6a a8 57 52 51 a2 e0 c0  c9 08 e1 1a 81 b8 0e 0e  |j.WRQ...........|
+000000f0  9c 4c 10 ae 31 88 9b e0  c0 f9 ff 3f 98 6b 02 e2  |.L..1......?.k..|
+00000100  02 00 01 31 13 36 0e 00  78 da 2b 67 80 00 49 28  |...1.6..x.+g..I(|
+00000110  0d 00 08 48 00 91 06 01  00 00 00 00 00 00 14 00  |...H............|
+00000120  78 da 6b 60 80 80 04 06  54 a0 00 a5 1d a0 34 00  |x.k`....T.....4.|
+00000130  24 28 01 41 1e 01 00 00  00 00 00 00 00 00 00 00  |$(.A............|
+00000140
+"""
+)
+
+SQUASHFS_V3_BE_NO_PAD_CONTENTS = unhex(
+    """\
+00000000  73 71 73 68 00 00 00 05  00 00 00 00 00 00 00 00  |sqsh............|
+00000010  00 00 00 00 00 00 00 00  00 00 00 00 00 03 00 01  |................|
+00000020  00 00 00 11 c0 01 00 62  03 cc c3 00 00 00 00 00  |.......b........|
+00000030  00 00 80 00 02 00 00 00  00 00 01 00 00 00 00 00  |................|
+00000040  00 00 00 00 00 01 40 00  00 00 00 00 00 01 3c 00  |......@.......<.|
+00000050  00 00 00 00 00 00 00 00  00 00 00 00 00 00 90 00  |................|
+00000060  00 00 00 00 00 00 cb 00  00 00 00 00 00 01 14 00  |................|
+00000070  00 00 00 00 00 01 34 78  da 4b 2c 28 c8 49 35 e4  |......4x.K,(.I5.|
+00000080  4a 04 51 46 10 ca 18 42  99 70 01 00 8b ee 09 3b  |J.QF...B.p.....;|
+00000090  00 39 78 da 53 5c c2 f0  3f 89 79 f3 45 06 06 06  |.9x.S\\..?.y.E...|
+000000a0  66 06 4c c0 ae 88 90 67  41 97 43 93 67 45 93 e7  |f.L....gA.C.gE..|
+000000b0  43 93 67 42 93 17 05 c9  0b fe 05 cb 5f 07 b2 19  |C.gB........_...|
+000000c0  c1 6a 38 e1 f2 6c 00 6f  a6 13 17 00 35 78 da 63  |.j8..l.o....5x.c|
+000000d0  66 80 00 66 06 26 4e 06  86 c4 82 82 9c 54 43 bd  |f..f.&N......TC.|
+000000e0  92 8a 12 46 20 97 11 cc  35 02 71 99 80 5c 26 30  |...F ...5.q..\\&0|
+000000f0  d7 18 c4 65 66 e2 fc ff  1f cc 35 01 71 01 cb 90  |...ef.....5.q...|
+00000100  11 84 80 10 00 00 00 00  00 00 00 77 00 00 00 19  |...........w....|
+00000110  00 00 00 00 00 00 00 00  00 00 01 02 00 16 78 da  |..............x.|
+00000120  63 60 00 83 06 08 c5 90  c0 80 0a 14 a0 b4 03 00  |c`..............|
+00000130  1b 68 01 41 00 00 00 00  00 00 01 1c 00 00 00 00  |.h.A............|
+00000140
+"""
+)
+
+
+def pad_contents(contents: bytes, pad_len: int):
+    return contents + b"\0" * (pad_len - len(contents))
```

Comment:
This only works if contents length is lower than pad_len. I would also use the term `alignment` rather than `pad_len`.

I would suggest something like this:

```suggestion
def pad_contents(contents: bytes, alignment: int):
    content_size = len(contents)
    padded_size = round_up(content_size, alignment)
    return contents + b"\0" * (padded_size - size)
```